### ISSUE TITLE
REQ: Use ending wildcard * to find/match channel names

### ISSUE THREAD

**Issue by rayholland** [beralt/horepg#14]
Let my try to explain my current problem:
I only want to stream SD channels, but some channels are not found/matched 

i.e. RTL 7

After some investigation it turns out that the horizon.tv API, only outputs for example the RTL 7 HD channel.
See https://web-api-pepper.horizon.tv/oesp/api/NL/nld/web/channels.com
":{"id":"24443943014","countryCode":"NL","locationId":"24443942973","title":"RTL 7 HD",".........

Would it be possible to find/match the channels, based on channel name PLUS a wildcard * ?
i.e RTL 7 -> RTL 7* (tvheadend channel RTL 7 would then be matched against horizon.tv RTL 7 HD epg data)

**Comment by beralt** [comment]
Yes, this used to be the case in the past, but that was removed in favor of having the stuff pulled from TVHeadend (see d8f9018). The code that does what you want is still there though. I'd rather have this configured from TVHeadend if possible. You can still do this by renaming your RTL 7 channel to RTL 7 HD; I think the latest version of TVHeadend has this specifically for EPG only purposes somewhere.

The reasoning behind the TVHeadend version is that, when you sync the DVB-C names in TVHeadend, it does not break the Horizon API interface because the provider uses that exact same link for the services that it provides. For that reason I would not like to implement the wildcard.

**Comment by RudolfRendier** [comment]
Here's what I did:

Create a new user in TVH for horepg, limit it to a label containing at least the HD channels.
Optionally hide this label from your regular users (so you limit them to SD channels)

Run horepg, pull the egp-data into TVH.

Configure the SD channels to use the EPG of its HD-counterpart.
